### PR TITLE
Install SNI related packages and requests with security option (#831)

### PR DESCRIPTION
**Pull request by whimboo**
Fix for issue #831. @mjzffr or @davehunt could one of you please review? Thanks.

### CODE REVIEW

File: jenkins-master/jobs/scripts/workspace/requirements.txt [hunk 1]
```diff
@@ -6,8 +6,14 @@ treeherder-client==2.0.1
 # Required by mozinfo
 mozfile==1.2
 
+# https://github.com/mozilla/mozmill-ci/issues/628 (and 831)
+# As long as Python 2.7.10 cannot be used on our master machines
+ndg-httpsclient
+pyasn1
+pyopenssl
```
Comment (whimboo) [inline jenkins-master/jobs/scripts/workspace/requirements.txt]: Those lines are just a copy from the other global requirements file.

Comment (davehunt) [inline jenkins-master/jobs/scripts/workspace/requirements.txt]: Which global requirements file? Should we pin to specific versions?

Comment (whimboo) [inline jenkins-master/jobs/scripts/workspace/requirements.txt]: The second requirements file which gets changed in this PR, and which already has those lines. I can pin the versions for sure.

Comment (whimboo) [inline jenkins-master/jobs/scripts/workspace/requirements.txt]: Dave, the PR has been updated for pinned versions which are proven to work on the CI master machines for a while.

**Pull request merged**